src/test/java/com/google/sps/data/friend_map/FriendsMapCreationTest.java

```diff
@@ -74,48 +71,52 @@ public void oneUserGraphTest() {
   */
   @Test
   public void twoUsersNoConnectionsGraphTest() {
-    userA = new UserNode(USER_A_ID, ImmutableSet.of());
-    userB = new UserNode(USER_B_ID, ImmutableSet.of());
+    UserNode userA = new UserNode(USER_A_ID, ImmutableSet.of());
+    UserNode userB = new UserNode(USER_B_ID, ImmutableSet.of());
 
     Set<UserNode> twoUserSet = new HashSet<>();
```

Comment:
If you want, this can be a little more compact:

```
Set<UserNode> twoUserSet = new HashSet<>(Arrarys.asList(userA, userB));
```

You could also use my favorite word - immutable! Your constructor doesn't modify the set, so you can safely do this because `ImmutableSet` implements the `Set` interface. For reference, `ImmutableSet` throws an `UnsupportedOperationException` on any method that would otherwise mutate the set (e.g., `add()`).

```
Set<UserNode> twoUserSet = ImmutableSet.of(userA, userB);
```

Reply:
Got it! I changed this to form an ImmutableSet.